webgui-new/next.config.js

```diff
@@ -15,6 +15,14 @@ const nextConfig = {
     assetPrefix: `${process.env.PUBLIC_URL || ''}/new`,
     basePath: `${process.env.PUBLIC_URL || ''}/new`,
   }),
+  async rewrites() {
+    return [
+      {
+        source: '/ga.txt',
+        destination: `${process.env.BACKEND_URL}/ga.txt`,
+      },
+    ];
+  },
```

Comment:
Other path rewrites for the backend are handled in `src/server.ts`. I suggest to handle this there as well.

Reply:
Fixed